inc/lift/RequestPool.h

```diff
@@ -68,6 +68,12 @@ class RequestPool
         std::chrono::milliseconds timeout
     ) -> Request;
 
+    auto Produce(
```

Comment:
does this need a full Produce() override or can it bet a default parameter or just set after the Produce() call is done?  this feels heavy

Reply:
Good point -- default parameter seems like the way to go here.